static/src/javascripts/projects/commercial/modules/prebid/bid-config.js

```diff
@@ -315,6 +315,91 @@ const getXaxisPlacementId = (sizes: PrebidSize[]): number => {
     return 13663304;
 };
 
+const getPangaeaPlacementId = (sizes: PrebidSize[]): number => {
+    type PangaeaSection = {
+        sections: Array<string>,
+        lb: number,
+        mmpu: number,
+        dmpu: number,
+    };
+    const pangaeaList: Array<PangaeaSection> = [
+        {
+            sections: ['business'],
+            lb: 13892359,
+            mmpu: 13892404,
+            dmpu: 13892360,
+        },
+        {
+            sections: ['culture'],
+            lb: 13892361,
+            mmpu: 13892405,
+            dmpu: 13892362,
+        },
+        {
+            sections: ['uk', 'us', 'au'],
+            lb: 13892363,
+            mmpu: 13892406,
+            dmpu: 13892364,
+        },
+        {
+            sections: ['news'],
+            lb: 13892365,
+            mmpu: 13892407,
+            dmpu: 13892366,
+        },
+        {
+            sections: ['money'],
+            lb: 13892367,
+            mmpu: 13892408,
+            dmpu: 13892368,
+        },
+        {
+            sections: ['sport'],
+            lb: 13892372,
+            mmpu: 13892410,
+            dmpu: 13892373,
+        },
+        {
+            sections: ['lifeandstyle', 'culture'],
+            lb: 13892411,
+            mmpu: 13892436,
+            dmpu: 13892437,
+        },
+        {
+            sections: ['technology', 'environment'],
+            lb: 13892376,
+            mmpu: 13892414,
+            dmpu: 13892377,
+        },
+        {
+            sections: ['travel'],
+            lb: 13892378,
+            mmpu: 13892415,
+            dmpu: 13892379,
+        },
+    ];
+
+    const section: string = (config.get('page.section') || '').toLowerCase();
```

Comment:
Not sure how I feel about this; I only did it for it to work in a test environment so should probably be stubbing it. However, maybe `page.section` sometimes is undefined?

Reply:
using `config.get('page.section', '')` should give the same result. 😁 Then you can safely call `.toLowerCase()`. Agree that section is likely never going to be intentionally `undefined`, however some other javascript somewhere may accidentally overwrite the config some day?

```js
// allows you to safely get items from config using a query of
// dot or bracket notation, with optional default fallback
const get = (path: string = '', defaultValue: any): any => {
```